config/locales/it.yml

```diff
@@ -0,0 +1,741 @@
+      storiessubmitted: "Articoli pubblicati:"
+      postedcomments: "Commenti pubblicati:"
+      mostcommonlytagged: "più comunemente etichettato"
+      about: "A riguardo:"
+      aboutdefault: "Un mistero..."
+      administrativeinfo: "Informazioni amministrative"
+      email: "E-mail:"
+      recentvotes: "Voti recenti:"
+      commentlinktext: "comment"
+      administrativeactions: "Azioni amministrative"
+      unbanuser: "Sbanna utente"
+      banningdescription: "Bannare o disabilitare gli inviti per un utente invierà una e/mail all'utente con la ragione indicata sotto, e il tuo indirizzo sarà presente nell'e-mail affinché l'utente possa replicare."
+      banningreason: "Ragione:"
+      banuserbutton: "Banna utente"
+      enableinvitesforuser: "Abilita gli inviti per l'utente"
+      disableinvitesreason: "Ragione:"
+      disableinvitesbutton: "Disattiva gli inviti"
+    tree:
+      newestusers: "Utenti più recenti"
+      administrator: "amministratore"
+      moderator: "moderatore"
+  tags:
+    index:
+      tagstitle: "Etichette"
+# data for users/invitationform which is a partial view
+  invitationtext: "Gli inviti sono illimitati, ma le persone da te invitate saranno associate al tuo profilo nell'<a href=\"/u\">albero degli utenti</a> e tu sarai responsabile per la loro condotta qualora queste causassero problemi. Per favore esercita accortezza nell'invitare persone che non conosci personalmente."
+  emailaddress: "Indirizzo e-mail:"
+  memouser: "Notifica all'utente:"
+  sendinvitation: "Manda un invito"
+  controllers:
+    comments_controller:
+      commentstitle: "Commenti più recenti"
+      threadstitle: "Le tue discussioni"
+    filters_controller:
+      filterstitle: "Filtri"
+      flashfilterupdate: "I tuoi filtri sono stati aggiornati."
+    hats_controller:
+      hatstitle: "Cappello"
+    home_controller:
+      recenttitle: "Articoli recenti"
+      abouttitle: "A riguardo"
+      chattitle: "Chat"
+      privacytitle: "Riservatezza"
+      hiddenstoriestitle: "Articoli nascosti"
+      rssforstoriestitle: "RSS 2.0"
+      rssforcommentstitle: "Commenti - RSS 2.0"
+      neweststoriestitle: "Articoli più nuovi"
+      licensetext: "<div class=\"box wide\">Toutes les actions sur le site sont publiques à l'exception des messages privés entre utilisateurs. Les actions de modération peuvent être consultées via le <a href=\"/moderations\">Journal de modération</a>.<br/><a rel=\"license\" href=\"https://creativecommons.org/licenses/by/4.0/deed.fr\"><img alt=\"Licence Creative Commons\" style=\"border-width:0\" src=\"https://i.creativecommons.org/l/by/4.0/88x31.png\" /></a><br /><span xmlns:dct=\"http://purl.org/dc/terms/\" property=\"dct:title\">Le Journal du hacker</span> est mis à disposition selon les termes de la <a rel=\"license\" href=\"https://creativecommons.org/licenses/by/4.0/deed.fr\">licence Creative Commons Attribution 4.0 International</a>.</div>"
```

Comment:
Questo alla fine si può tradurre così com'è

Reply:
Lo traduciamo con ausilio di deepL o fai tu?

Reply:
mergiamo in francese, poi lo traduco nella mia PR